package.json

```diff
@@ -43,7 +45,7 @@
     "postinstall": "node ./scripts/postinstall.js && node ./scripts/build.js",
     "publish": "yarn run build-clean && yarn run build && lerna publish",
     "test": "yarn run typecheck && yarn run lint && yarn run build && yarn run jest && yarn run test-examples",
-    "test-ci": "yarn run typecheck && yarn run lint && yarn run build && yarn run jest-coverage -- -i && yarn run test-examples && node scripts/mapCoverage.js && codecov",
+    "test-ci": "yarn run typecheck && yarn run lint && yarn run build && yarn run jest-coverage -- -i && yarn run test-examples && node scripts/mapCoverage.js && codecov && yarn run danger",
```

Comment:
I assume codecov reads some pre-existing files to derive output? It's totally feasible ( we have ruby Danger parsing JUnit files in our iOS app ) 

However, as you noted below, it's the state that doesn't necessarily work. In ruby Danger, there is a plugin for persistence (https://github.com/dbgrandi/danger-mcbrain) which could get replicated as a free heroku mongo somewhere or something to keep state